packages/core/package.json

```diff
@@ -172,8 +161,6 @@
     "@formatjs/intl-relativetimeformat": "9.3.1",
     "@hyperledger/anoncreds-react-native": "0.2.4",
     "@hyperledger/aries-askar-react-native": "0.2.3",
-    "@hyperledger/aries-bifold-verifier": "*",
-    "@hyperledger/aries-oca": "*",
```

Comment:
It's very possible to use core without having these deps installed in your project, so I removed them as peers